### ISSUE TITLE
Add support for remote play/pause event listener

### ISSUE THREAD

**Issue by gazedash** [tanguyantoine/react-native-music-control#233]
# Description

It would be really useful to add listeners for remote play/pause actions
E.g. bluetooth headphones / Apple TV

https://github.com/tanguyantoine/react-native-music-control/issues/123

2. Platform ?

   - [x] iOS
   - [ ] Android
   
   
3. Device
  - [ ] Simulator
  - [x] Real device

**Comment by tanguyantoine** [comment]
Can you elaborate? Can you provide any links to help make this feature move forward?

**Comment by gazedash** [comment]
@tanguyantoine it seems like it is related to togglePlayPauseCommand 
https://stackoverflow.com/a/32373337/7318811

I have enabled togglePlayPause and I have also registered togglePlayPause:

```
MusicControl.on('togglePlayPause', ()=> {
      alert('toggle!')
    });
```

But it doesn't work. Other events work though, e.g. if I press play / pause button in my app, it is called.

For more info, I am using react-native-video library
